content/rest-api/beta.textile

```diff
@@ -0,0 +1,165 @@
+---
+title: REST API Beta Features
+section: rest-api
+index: 0
+api_separator:
+jump_to:
+  Help with:
+    - Overview#overview
+    - Batch Publish#batch
+---
+
+h2(#overview). REST API Beta Features
+
+The contents of this section are beta features of our REST API. Although many of these features are fully functional, they are liable to change.
```

Comment:
Perhaps state instead that until the feature is released as generally available, the APIs may change.  We recommend customers get in touch if they want to use these features in production.